tests/conftest.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from app import create_app
+from app.db import db
+from flask.signals import request_finished
+import os
```

Comment:
Looking at the next PR which has a test, I'm getting a failure that the connection string isn't set. It doesn't appear, for me at least, that the environment is loaded automatically by flask in time for the test connection string to be available. I had to manually add the following to get this to load properly.

```suggestion
from dotenv import load_dotenv
import os

load_dotenv()
```

It's a little inelegant, but I'm not sure what else to do. Another option would be to have the test case set not the connection string itself, but the name of the key to use for the connection string. But that probably muddies the waters even more?

Reply:
I'm good adding this in case other folks run into it and also prefer the suggested solution here over setting the name of the key for the connection string. 

Do you have further details on how you ran into the test failure? I can't reproduce the issue on my end after checking that I don't have any of these keys accidentally set in my global environment and deleting and re-cloning the repo to make sure I didn't have anything extra hanging around.

Reply:
I don't see any error until the 6c branch. Since the 6b branch had no tests, nothing ever tried to create the app. But when I checked out the 6c branch, running `pytest` dumped a large stack trace that ultimately was complaining that `SQLALCHEMY_DATABASE_URI` wasn't set.

Reply:
I made sure to grab the 6c branch based on your comments, but I may have only been running the tests from VSCode, so I'll try from Terminal.

Reply:
Yep, that was it, I never run into it from VSCode, but it happens in Terminal. I'll get the branches updated shortly =]

Reply:
Oh. That was super weird. Yeah. From in VSCode (even VSCode terminal!) it all passed! 😱